src/content/docs/features/cachyos_settings.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: CachyOS Settings
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+
+Alongside our optimized kernels and repositories, we also provide settings that further improve the desktop experience.
+We also bundle in some helper scripts to for QoL improvements. All these configurations and scripts are under the `cachyos-settings` package.
+
+## sysctl Tweaks
+
+We provide a lot of sysctl tweaks that aims to improve overall desktop performance. Each sysctl entry is documented
+well in the file [`99-cachyos-settings.conf`](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/sysctl.d/99-cachyos-settings.conf)
+
+To make changes to any of these values, please copy the original file to `/etc/sysctl.d` and modify it from there
+
+```sh title="Copying 99-cachyos-settings.conf to /etc/sysctl.d"
+sudo cp /usr/lib/sysctl.d/99-cachyos-settings.conf /etc/sysctl.d/
+```
+
+## udev Rules
+
+- [ZRAM Rules](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/30-zram.rules) - Sets ZRAM swappiness to a more aggressive
+value so cache is more likely to swap to ZRAM
+- [HPET Permissions](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/40-hpet-permissions.rules) - Lower latency audio
+- [SATA Power Management](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/50-sata.rules) - Sets power management policy of SATA devices to `max_performance`
+- [I/O Scheduler Rules](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/60-ioschedulers.rules) - Selects the optimal scheduler for each drive type (HDD, SSD, NVME)
+- [hdparm Rules](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/69-hdparm.rules) - Sets SATA and IDE HDDs to maximum performance
+- [NVIDIA RTD3](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/71-nvidia.rules) - Fully powers down
+NVIDIA GPUs when not in use
+- [CPU DMA Latency](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/99-cpu-dma-latency.rules) -
+Avoids entering Deeper C-States when playing audio to reduce latency
+- [NTSync](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/99-ntsync.rules) - Enables NTSync
```

Comment:
Same as with cpu_dma_latency. This just sets the correct permissions so that dev node can be accessed by user-space applications. It doesn't enable NTSync itself.